{"uuid":"ef7ed405-2123-40f6-82e2-7782d29759e1","historyId":"4f372b2da250dd3429ba454ecbee72a6","fullName":"src/test/resources/features/User.feature:96","labels":[{"name":"host","value":"DESKTOP-AKTBA7H"},{"name":"thread","value":"60968@DESKTOP-AKTBA7H.main(1)"},{"name":"feature","value":"Validating User API"},{"name":"story","value":"Check if user able to update with invalidrequestbody"},{"name":"suite","value":"Validating User API"},{"name":"testClass","value":"Check if user able to update with invalidrequestbody"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/rajas/git/repository/UserBDD_Bootcamp/src/test/resources/features/User.feature"},{"name":"package","value":"src.test.resources.features.User_feature.Validating User API"}],"links":[],"name":"Check if user able to update with invalidrequestbody","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected:<40[4 NOT_FOUND]> but was:<40[0 BAD_REQUEST]>","trace":"org.junit.ComparisonFailure: expected:<40[4 NOT_FOUND]> but was:<40[0 BAD_REQUEST]>\r\n\tat org.junit.Assert.assertEquals(Assert.java:117)\r\n\tat org.junit.Assert.assertEquals(Assert.java:146)\r\n\tat stepDefinition.UserSteps.user_receives_with_message(UserSteps.java:196)\r\n\tat ✽.user receives \"400 BAD_REQUEST\" with message \"Contact number is already in use by another user\"(file:///C:/Users/rajas/git/repository/UserBDD_Bootcamp/src/test/resources/features/User.feature:99)\r\n"},"stage":"finished","steps":[{"name":"Given  user with Invalid payload","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1716505780811,"stop":1716505780819},{"name":"When  user send HTTPS Request and request Body with invalid data","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1716505780819,"stop":1716505781267},{"name":"Then  user receives \"400 BAD_REQUEST\" with message \"Contact number is already in use by another user\"","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected:<40[4 NOT_FOUND]> but was:<40[0 BAD_REQUEST]>","trace":"org.junit.ComparisonFailure: expected:<40[4 NOT_FOUND]> but was:<40[0 BAD_REQUEST]>\r\n\tat org.junit.Assert.assertEquals(Assert.java:117)\r\n\tat org.junit.Assert.assertEquals(Assert.java:146)\r\n\tat stepDefinition.UserSteps.user_receives_with_message(UserSteps.java:196)\r\n\tat ✽.user receives \"400 BAD_REQUEST\" with message \"Contact number is already in use by another user\"(file:///C:/Users/rajas/git/repository/UserBDD_Bootcamp/src/test/resources/features/User.feature:99)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1716505781269,"stop":1716505781345}],"attachments":[],"parameters":[],"start":1716505780811,"stop":1716505781359}